test/unit/spec/v2/OktaSignInWidget_spec.js

```diff
@@ -36,20 +25,32 @@ Expect.describe('OktaSignIn initialization', function () {
       }
     });
     spyOn(signIn.authClient.tx, 'introspect').and.callFake(function () {
-      return options.response;
+      return Q(introspectResponse.response);
     });
-    spyOn($, 'ajax');
+  });
+
+  afterEach(function () {
+    signIn.remove();
+  });
+
+  function setupIntrospect () {
     signIn.renderEl({ el: $sandbox });
-    return Q({});
+    return Expect.waitForSpyCall(viewFactory.create);
   }
-  Expect.describe('Introspects token on load', function () {
+  Expect.describe('Introspects token and loads Identifier view for new pipeline', function () {
     it('calls introspect API on page load', function () {
       return setupIntrospect({
         response: {
           version: '1.0.0'
         }
       }).then(function () {
+        //TODO replace with idx js mock
```

Comment:
I think you wont do idx integration in PR, correct?

Reply:
yes, but this is just a note for later